distributed/worker.py

```diff
@@ -51,6 +51,8 @@
     typename,
 )
 
+from distributed.compatibility import to_thread
```

Comment:
I'll move to relative import for consistency in this file. the overall code base is pretty inconsistent, though. I think we should settle this https://github.com/dask/distributed/issues/5889 since imo we should not debate style issues on PRs